chapter-07/wip/Program.cs

```diff
@@ -14,7 +16,7 @@ public static class Program
         [STAThread]
         static void Main()
         {
-            using (var game = new MainGame())
+            using (var game = new MainGame(1280, 720, new SplashState()))
```

Comment:
I would still move these magic numbers to a constant even if it's just within this file